shared/js/background/click-to-load.js

```diff
@@ -0,0 +1,69 @@
+import tdsStorage from './storage/tds'
+import { sendTabMessage } from './utils'
+
+/**
+ * Find the enabled Click to Load rule actions for the given tab.
+ * Note: Take care to ensure wait for the extension configuration to be ready
+ *       first.
+ * @param {import('./classes/tab')} tab
+ * @return {string[]}
+ */
+export function getDefaultEnabledClickToLoadRuleActionsForTab (tab) {
```

Comment:
(As Sam pointed out, this function is not declarativeNetRequest specific, so it made sense to split it out of the dnr-click-to-load.js module.)